assets/js/theme/common/product-details.js

```diff
@@ -25,6 +25,21 @@ export default class ProductDetails extends ProductDetailsBase {
         const hasOptions = $productOptionsElement.html().trim().length;
         const hasDefaultOptions = $productOptionsElement.find('[data-default]').length;
         const $productSwatchGroup = $('[id*="attribute_swatch"]', $form);
+        const $productSwatchLabels = $('.form-option-swatch', $form);
+        const placeSwatchLabelImage = (_, label) => {
+            const $optionImage = $('.form-option-expanded', $(label));
+            const optionImageWidth = $optionImage.outerWidth();
+            const extendedOptionImageOffsetLeft = 55;
```

Comment:
can we replace this static value as some constant above to not reinit this variable for each swatch option item?

Reply:
good point, but if you don't mind I let it be as it is - the variable i belongs to the placeSwatchLabelImage function and won't be used elsewhere so I find it lexically more suitable to place it inside function.